scripts/score.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+import json
+import os
+import sys
+from typing import Any, Optional
```

Comment:
Do we need `Optional` (do we just use the `|` usually elsewhere?)

Reply:
no – removed